src/ucp/wireup/address.h

```diff
@@ -106,17 +109,19 @@ typedef ucs_bitmap_t(UCP_MAX_RESOURCES) ucp_tl_addr_bitmap_t;
  * Remote interface attributes.
  */
 struct ucp_address_iface_attr {
-    uint64_t                    cap_flags;    /* Interface capability flags */
-    uint64_t                    event_flags;  /* Interface event capability flags */
+    uint64_t                    flags;        /* Interface capability and event
+                                                 flags */
     double                      overhead;     /* Interface performance - overhead */
     double                      bandwidth;    /* Interface performance - bandwidth */
     int                         priority;     /* Priority of device */
-    double                      lat_ovh;      /* Latency overhead */
+    double                      lat_ovh;      /* Address v1: latency overhead
```

Comment:
maybe rename to latency
and with v1 it will calculate num_eps during address unpack
then, could remove addr_version field

Reply:
when can we take `latency.m` component during address unpack?

Reply:
ok